### ISSUE TITLE
Get the Tensorboard Graphs

### ISSUE THREAD

**Issue by Darshvino** [feiyuhuahuo/Yolact_minimal#52]
Hi @feiyuhuahuo,

I am training the Yolact_minimal model on 4Million Images using 2 A100 Machines. 
These are the following Configs I have kept:

BS= 16
lr_steps= 800000
No of classes= 37
Val interval= 5000
Val Images(Among Train Set) = 30000
Backbone = Resnet101
And all other configs and params are the same as is.

Command used:  **python3 -m torch.distributed.launch --nproc_per_node=2 --master_port=$((RANDOM)) train.py --train_bs=16**

My Queries are:
1.) Whether all the above Configs are good and fine?
2.) How do I get the Tensorboard Metrics Graphs?
3.) How to resume the training on the last best.pth?

Please suggest if anything changes required, so that I will follow the same for the next round of training.

Thanks and Regards

**Comment by feiyuhuahuo** [comment]
4 million images are quite a large dataset and will take a long time to train. You should use as larger batch size as possible since A100 has large GPU memory. And since you have so many train images, why don't you split part of them to validate. Validating on train set is not a common way. About total training steps, it differs from dataset. You should train first and watch the val accuracy. If the val accuracy no longer increase, then decay the lr.  
Resume training: python -m torch.distributed.launch --nproc_per_node=1 --master_port=$((RANDOM)) train.py --resume=weights/latest_res101_coco_35000.pth  
Tensorboard: tensorboard --logdir=tensorboard_log/res101_coco

**Comment by Darshvino** [comment]
Hi @feiyuhuahuo,

Thank you very very much for your kind support.

I trained the model for around 230k steps and below are the training curves:

![image](https://user-images.githubusercontent.com/41161376/135169803-33963e42-c022-4801-ac6c-9587eb2495a1.png)

![image](https://user-images.githubusercontent.com/41161376/135169825-1b642d3c-6d02-4cdd-8666-9cf08a3bed2a.png)

How do we get the validation curves? and everything looks good about the loss and the mAP? 

Thanks and Regards

**Comment by feiyuhuahuo** [comment]
The mAP curves are on validation dataset. Everything looks ok.

**Comment by Darshvino** [comment]
Hi @feiyuhuahuo,

I have trained the model for around 230K Steps. And I have run the inference on the trained model, but the results seem to be not good. Not Sure what is the issue. Please find the attached link for the Inference results: https://drive.google.com/drive/folders/1_Krpbj5PxYpsFVdpjKSn3kmCZQSiXikz?usp=sharing

Whether there is any specific reason for the bad results? I have trained on 544 input size and I have given 1028x1232 image for inference. Please help me to resolve the issue. Would be a great help from your side.

Looking forward to your reply.

Thanks and Regards

**Comment by Darshvino** [comment]
Hi @feiyuhuahuo,

Looking forward to your reply :)

Thank you

**Comment by feiyuhuahuo** [comment]
Try to use 544 img size for inference,   1028x1232 is too difference from the train size. And these days I can't get access to google drive.
